config/G00_00b/ModelConfiguration.xml

```diff
@@ -139,6 +139,7 @@ STFC, Rutherford Appleton Laboratory
   <param type="alg" name="XSecModel@genie::EventGenerator/MEC-EM">         genie::EmpiricalMECPXSec2015/Default  </param>
   <param type="alg" name="XSecModel@genie::EventGenerator/NucleonDecay">   genie::DummyPXSec/Default             </param>
   <param type="alg" name="XSecModel@genie::EventGenerator/NNBarOsc">       genie::NNBarOscDummyPXSec/Default     </param>
+  <param type="alg" name="XSecModel@genie::EventGenerator/BeamHNL"> genie::DummyHNLPXSec/Default  </param>
```

Comment:
HNL does have its own tune `GHNL20_00a_00_000`, but without the `--tune` argument it turns out `RunOpt` was configuring the default GENIE tune `G18_02a_00_000`. 
The desired outcome is to point to GHNL20_00a_00_000 without needing to specify `--tune`, but also allow the user a `--tune` argument in case of future HNL tunes.
This outcome has now been achieved through adding in a `--tune GHNL20_00a_00_000` bit to what `RunOpt` sees upon getting common Options.

Reply:
All references to the `genie::EventGenerator/BeamHNL` Algorithm have now been removed from the SM GENIE tunes.